Library/Homebrew/extend/os/mac/development_tools.rb

```diff
@@ -9,7 +9,7 @@ def locate(tool)
         @locate[key] = if (located_tool = original_locate(tool))
           located_tool
         elsif MacOS.version > :tiger
-          path = Utils.popen_read("/usr/bin/xcrun", "-no-cache", "-find", tool).chomp
+          path = Utils.popen_read("/usr/bin/xcrun", "-no-cache", "-find", tool, err: :close).chomp
```

Comment:
You can probably remove these two now in favour of just the `HOMEBREW_STDERR` below, I think?

Reply:
I plan on enabling `HOMEBREW_STDERR=1` globally for myself. These two commits silence that wee bit of unnecessary noise from `brew config`. Happy to drop them though if you prefer.